sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/storage/AuthTokenStorage.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.appcenter.identity.storage;
+
+/**
+ * Interface for storage that works with token.
+ */
+public interface AuthTokenStorage {
+
+    /**
+     * Stores token value.
+     *
+     * @param token auth token.
+     * @param homeAccountId unique identifier of user.
+     */
+    void saveToken(String token, String homeAccountId);
```

Comment:
Similarly to @jwallra's [comment](https://github.com/Microsoft/AppCenter-SDK-Android/pull/956/files#r258206045), I think the `saveToken` naming is confusing because it takes 2 parameters: token value and account id value.

Reply:
Same confusion for `cacheToken` naming. It caches token value and account id value.

Reply:
Ignore my first 2 comments. I realized account id is actually a claim from the auth token, which means `homeAccountId` is a part of the `token`, therefore `AuthTokenStorage` is still the right naming.